pkg/ctl/create/cluster.go

```diff
@@ -413,6 +401,27 @@ func doCreateCluster(cmd *cmdutils.Cmd, ngFilter *filter.NodeGroupFilter, params
 	return printer.LogObj(logger.Debug, "cfg.json = \\\n%s\n", cfg)
 }
 
+// installKarpenter prepares the environment for Karpenter, by creating the following resources:
+// - iam roles and profiles
+// - service account
+// - identity mapping
+// then proceeds with installing Karpenter using Helm.
+func installKarpenter(ctl *eks.ClusterProvider, cfg *api.ClusterConfig, stackManager manager.StackManager, clientSet *kubeclient.Clientset) error {
+	if cfg.Karpenter == nil {
+		return nil
+	}
```

Comment:
do we validate elsewhere `.version` is set?

Reply:
Like, that it's a required field?

Reply:
Huh, guess I don't check it. :O

Reply:
:smile:

Reply:
Added :D